From: Ben Hutchings <ben@decadent.org.uk>
Date: Thu, 16 Jan 2014 08:11:20 +0000
Subject: mm/migrate: Avoid ABI change in 3.12.7

genksyms is confused by extra definitions.  Hide the new #include.

--- a/mm/migrate.c
+++ b/mm/migrate.c
@@ -36,7 +36,9 @@
 #include <linux/hugetlb_cgroup.h>
 #include <linux/gfp.h>
 #include <linux/balloon_compaction.h>
+#ifndef __GENKSYMS__
 #include <linux/mmu_notifier.h>
+#endif
 
 #include <asm/tlbflush.h>
 
